ci_support/run_docker_build.sh

```diff
@@ -37,8 +37,6 @@ echo "$config" > ~/.condarc
 # A lock sometimes occurs with incomplete builds. The lock file is stored in build_artefacts.
 conda clean --lock
 
-# library dependencies for matplotlib
-yum install -y libXext libXrender libSM tk libX11-devel
```

Comment:
I think we added these right? In any event, we can add yum requirements too as needed at this point. Maybe its time for a fresh re-rendering PR here.